Resources/docs/adapters/awss3.md

```diff
@@ -1,8 +1,33 @@
-# Amazon S3 SDK v2
+# AWS S3
 
-Adapter for Amazon S3 SDK v2.
```

Comment:
Can you add a table of content referencing the three sections of this file please ?
```markdown
- [Amazon SDK v3 (recommended)](#adapter-for-amazon-s3-sdk-v3-recommended)
- [Amazon SDK v2](#adapter-for-amazon-s3-sdk-v2)
- [Amazon SDK v1 (deprecated)](#adapter-for-amazon-s3-sdk-v1-deprecated-by-amazon)
```